.gitignore

```diff
@@ -3,6 +3,7 @@
 .DS_Store
 .idea
 *.pki
+**/.sqlx
```

Comment:
Hmm... Why is this here? AFAIK you need these files in order to be able to compile the code that uses `sqlx`. These also need to be updated regularly when changing DB (though the build usually fails when there's need).

See https://github.com/launchbadge/sqlx/?tab=readme-ov-file#compile-time-verification and https://github.com/launchbadge/sqlx/blob/main/sqlx-cli/README.md#enable-building-in-offline-mode-with-query

Reply:
I thought it was sqlx temporary file folder. I've one deleted by sqlx, so I thought that it's a working directory of sqlx and not part of the code. I remove.